docs/fieldnorm/requirements_configuration.md

```diff
@@ -0,0 +1,28 @@
+# Requirements and configuration management
+
+The complexity and variety of developments in the field of bio-informatics, comes with a need for clear prerequisites before the start of a project on the one hand and concise regulation of development versioning on the other. Unfortunately, neither is described for application to medical devices by the ISO-15189, unlike the IEC-62304, which does disclose types of requirements that should be documented.
```

Comment:
bioinformatics in english ;). (skip the -)

The "unfortunately ISO15189 " is what we say in the introduction of the document, so you can remove the second line.

Reply:
The complexity and variety of developments in the field of bio-informatics, comes with a need for clear prerequisites before the start of a project on the one hand and concise regulation of development versioning on the other. 
It's a very long sentence! 
rephrase to (for example): 
To tackle the complexity of developing bioinformatic workflows clear prerequisites  are needed. These requirements are needed before the start of a project, and additionally when developing new features. 
(I don't understand the 'regulation of development versioning' here, could be me ;) )

Reply:
or did you want to try to make 1 sentence to cover both requirements and configuration management???  
I think these should become 2 different entries. As both were small, they became 1 task to do.